### PR TITLE
feat: Add emoji picker for comments

### PR DESCRIPTION
**Pull request by tokitou-san**
### CHANGELOG:
 - Added [`svelte-emoji-picker`](https://github.com/ehne/svelte-emoji-picker) lib for emoji picker
 - Added [`emoji-picker-element`](https://github.com/nolanlawson/emoji-picker-element) instead
 - Integrate emoji_picker with tippy ( as popover )
 - Added custom styles for emoji-picker

### CODE REVIEW

**Comment by tokitou-san** [comment]
@baseplate-admin 
It seems like `svelte-emoji-picker` has no typescript support yet.
So Imma create ourown for now

**Comment by baseplate-admin** [comment]
@tokitou-san, forget this thing. Use the library i mentioned in telegram

File: src/lib/components/shared/tippies/emoji_picker.svelte [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        --button-active-background: #4d4f68;
+        --input-border-color: #72717c;
+        --input-border-radius: 0.5vw;
+        --input-border-size: 0.1vw;
+        --outline-size: 0.1vw;
+        --outline-color: #9c9ba3;
+        --input-font-size: 1vw;
+        --input-padding: 0.35vw 0.75vw;
+    }
+</style>
```
Comment (baseplate-admin) [inline src/lib/components/shared/tippies/emoji_picker.svelte:42]: This is a very good engineering example. Well done!!

Comment (tokitou-san) [reply src/lib/components/shared/tippies/emoji_picker.svelte:42]: Thanks bud :)

**Pull request merged**